cfme/web_ui/__init__.py

```diff
@@ -2039,6 +2039,15 @@ def name(self):
         """ Returns name of the quadicon."""
         return self._name
 
+    @property
```

Comment:
Aziza, in the doc string, can you change to something closer to...
Checks if the quad icon is a single quadrent icon which is present after changing to tile(?) view.

And lets change the name too to check_for_single_quadrent_icon, I believe that should clear it up